src/api/routes/articleRouter.js

```diff
@@ -32,4 +33,8 @@ articleRouter.post('/',
   checkValidToken,
   uploadImage.single('coverImage'),
   errorHandler(articleController.create));
+
+articleRouter.post('/:articleId/highlight', checkValidToken, validateInputs(true, 'highlight', highlightFields),
```

Comment:
@alainburindi I think we need to check if the user is authorized

Reply:
@nkpremices isn't it what he is trying to do with the CheckValidToken middleware

Reply:
@alainburindi what happens if the article doesn't exist?